test/cmd/whoami.sh

```diff
@@ -12,7 +12,7 @@ trap os::test::junit::reconcile_output EXIT
 
 os::test::junit::declare_suite_start "cmd/whoami"
 # This test validates the whoami command's --show-server flag
-os::cmd::expect_success_and_text 'oc whoami --show-server' 'http(s)?:\/\/[0-9\.]+\:[0-9]+'
+os::cmd::expect_success_and_text 'oc whoami --show-server' 'http(s)?:\/\/localhost\:[0-9]+'
```

Comment:
Wait so this only ever runs locally now? Wasn't the point that you could run `test-cmd` against any cluster, too?

Reply:
> Wait so this only ever runs locally now? Wasn't the point that you could run `test-cmd` against any cluster, too?

We'll live with it to kill openshift start.  If someone wants it later, they can add it back.